src/Features/Core/Portable/ExternalAccess/UnitTesting/SolutionCrawler/IUnitTestingIncrementalAnalyzerProvider.cs

```diff
@@ -6,6 +6,10 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.SolutionCrawler
 {
     internal interface IUnitTestingIncrementalAnalyzerProvider
     {
-        IUnitTestingIncrementalAnalyzer? CreateIncrementalAnalyzer(Workspace workspace);
+        IUnitTestingIncrementalAnalyzer? CreateIncrementalAnalyzer(
+#if false // Not used in unit testing crawling
+            Workspace workspace
+#endif
```

Comment:
i have validated that unit testing does not use this workspace: http://index/?rightProject=Microsoft.CodeAnalysis.Features&file=ExternalAccess%5cUnitTesting%5cAPI%5cUnitTestingIncrementalAnalyzerProvider.cs&line=26

Reply:
This is also a general pattern i'm using in the fork.  Instead of deleting, i'm IFDEF'ing out so we can see how this drifts from teh original.

Reply:
Wouldn't simple diff of the files show that?

Reply:
possibly... though i worry about the diffs getting too far apart (and things like naming confusing it).  I'm flexible on this htough.  if we prefer just deleting, that's also ok with me.